packages/errors/src/react/button.js

```diff
@@ -1,21 +0,0 @@
-import buttonCss from '@pluralsight/ps-design-system-button/css'
-import * as buttonVars from '@pluralsight/ps-design-system-button/vars'
-import * as glamor from 'glamor'
-import React from 'react'
-
-const styles = {
-  button: _ =>
-    glamor.css(
-      buttonCss['.psds-button'],
-      buttonCss[`.psds-button--size-${buttonVars.sizes.medium}`],
-      buttonCss[`.psds-button--appearance-${buttonVars.appearances.primary}`],
-      {
-        ':hover': buttonCss['.psds-button:hover'],
-        ':focus': buttonCss['.psds-button:focus'],
-        ':focus:before': buttonCss['.psds-button:focus:before'],
-        ':focus:after': buttonCss['.psds-button:focus:after']
-      }
-    )
-}
-
-export default props => <button {...styles.button(props)} {...props} />
```

Comment:
this didn't support the `href` prop which the error component was supplying. i decided just to use the more common <Button /> component...it was already a package dep